packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/static-shadow/exec.js

```diff
@@ -0,0 +1,15 @@
+class Test {
+
+  static #x = 1
+
+  static method() {
+    const Test = 2;
+    const func = () => {
+      const Test = 3;
+      return this.#x;
+    }
+    return func();
```

Comment:
Can you also add some references to `Test`?
```suggestion
    const Test = 2;
    const func = () => {
      const Test = 3;
      return this.#x + Test;
    }
    return func() + Test;
```